### PR TITLE
Confetti Overlay improvements

### PR DESCRIPTION
**Pull request by jantocv**
## Description
 - Less number of particles to ensure good performance is achieved (80 by default are more than OK)
 - Validation for props
 - You can now pass string/number as particles
 - position properties changes.

## Testing
 - Unit tested
 - Storybook

## Screenshots/Captures

confett

## Checks
- [ ] Requires documentation update
- [X] Requires lib version update

## Versioning impact
- [ ] **MAJOR** incompatible API changes.
- [X] **MINOR** adds functionality in a backwards-compatible manner.
- [ ] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

File: src/components/ConfettiOverlay/ConfettiOverlay.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const maximumCountOfParticlesForSmoothPerformance = 200;
+
+const props = {
+  particles: {
+    type: [Number, String],
+    default: defaultCountOfParticles,
+    validator: (value) => {
+      const numericValue = Number.parseInt(value, 10);
+      const isValid = !Number.isNaN(numericValue);
+      return (isValid && numericValue <= maximumCountOfParticlesForSmoothPerformance);
```
Comment (jasonadt) [inline src/components/ConfettiOverlay/ConfettiOverlay.js:11]: (nit) please update this with a simple return, and enclose the second clause within parens, like so: 

```js
const result = (isValid && (numericValue <= maximumCountOfParticlesForSmoothPerformance));
return result;
```

File: src/components/ConfettiOverlay/ConfettiOverlay.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+  },
+};
+
+const computed = {
+  amountOfParticles() {
```
Comment (jasonadt) [inline src/components/ConfettiOverlay/ConfettiOverlay.js:17]: (nit) please rename this more naturally to: `countOfParticles`

**Review by jasonadt — CHANGES_REQUESTED** [review]

**Review by rahultallridge — DISMISSED** [review]

**Review by jasonadt — APPROVED** [review]
👍  thanks for these improvements! Great work! 

Ship it! 🎉

**Comment by jantocv** [comment]
@jasonadt  1 simple confetti component for me... 2 exhausting days for mankind...

**Pull request merged**